client-core/src/service/key_service.rs

```diff
@@ -56,7 +92,152 @@ where
 
     /// Clears all storage
     pub fn clear(&self) -> Result<()> {
-        self.storage.clear(KEYSPACE)
+        self.storage.clear(KEYSPACE).expect("clear keyspace");
+        self.storage
+            .clear(KEYSPACE_HD)
+            .expect("clear keyspace for hd");
+        Ok(())
+    }
+
+    /// get random mnemonic
+    pub fn get_random_mnemonic(&self) -> String {
```

Comment:
This function does not belong to `KeyService`.

Reply:
ok